src/composables/useNavigationGuards.ts

```diff
@@ -1,91 +1,24 @@
 import NProgress from 'nprogress';
```

Comment:
I've moved this logic into a higher-level nav-gaurds.ts file that is applied directly to the router since these redirects don't require app context.